packages/webapp/src/components/ChartSection/Markup.jsx

```diff
@@ -41,8 +67,14 @@ const callButtonExplore = (url, color, verb, subject) => {
   );
 };
 
-const callDetails = (selected, verb, subject) => {
-  const { name, value, url, color } = selected;
+const callAmount = (value, loading) => (
```

Comment:
what do we mean when we call these functions `call...`? It doesn't make sense to me as an old fart. You _call_ a function - it's weird to call `callFoo`. This function does call something (like most functions) - it calls the constructor of Amount - would it make more sense to call it `createAmount`? `constructAmount` might sound a bit like it's a factory for `Amount`s. 

In fact it creates Amount and decides what content should be inside it.

Is calling it `call...` a common pattern? I'm not saying it has to be changed, it's just sounds like we're writing a function to call a function and we call it callfunction.